### ISSUE TITLE
How does the jib jar specify the image deployment path and add a build file

### ISSUE THREAD

**Issue by gluo1130** [GoogleContainerTools/jib#3659]
**Environment**:

- *Jib version:*  0.10.0
- *Build tool:* jib jar 
- *OS:* MacOs、Linux

**Description of the issue**:
In the daily production environment, besides a jar, there are other additional files in the business. How can jib jar support the construction of other files as images. In addition, jib war can specify the image deployment path through **--app-root** support, but jib jar does not see the corresponding configuration. If you want to specify directory deployment, how to implement it

**Expected behavior**:
Jib jar supports building images from configuration files other than jar and specifying deployment paths. For example, I can specify the deployment path as **/home/work/appName/runtime** through jib jar and start the command **java -cp /app org .springframework.boot.loader.Jarlauncher** writes In the **start.sh** script, the entrypoint is specified as sh start.sh start image. Therefore, start.sh needs to be packaged and built as part of the image.

```
bash-4.1$ pwd
/home/work/hello-focus/runtime/app
bash-4.1$
bash-4.1$
bash-4.1$ ls
BOOT-INF  META-INF  log  org
```

**Steps to reproduce**:

  1. jib jar --target=docker://iregistry.baidu-int.com/meg-cprd-testmain/demo:0.1 --allow-insecure-registries=true --from=iregistry.baidu-int.com/meg-cprd-testmain/centos7u5/emcjdk8:0.1 --from-username=*** --from-password=***  demo-0.0.1-SNAPSHOT.jar

**Comment by gluo1130** [comment]
@mpeddada1 @chanseokoh

**Comment by elefeint** [comment]
@gluo1130 The [`jib build`](https://github.com/GoogleContainerTools/jib/tree/master/jib-cli#build-command) command will likely be more appropriate to your usecase -- it gives you more flexibility with which layers are built.

**Comment by gluo1130** [comment]
> @gluo1130 The [`jib build`](https://github.com/GoogleContainerTools/jib/tree/master/jib-cli#build-command) command will likely be more appropriate to your usecase -- it gives you more flexibility with which layers are built.

Yes,` jib build `can solve this problem, but we hope not to use fat jar as a layer for image construction. We can use the ability of jib jar to divide the layer of fat jar into multiple layers for image construction. After looking at the source code of jib cli, jib jar should not be supported at present. After receiving your reply, I wonder if you can solve this problem by directly decompressing the fat jar to the deployment path and using jib build to build the image.

**Comment by gluo1130** [comment]
>

**Comment by gluo1130** [comment]
> Yes, you can create the same outcome that `jib jar` does with `jib build` and additional manipulation of build artifacts and layers ([here is what `jib jar` does](https://github.com/GoogleContainerTools/jib/blob/master/docs/faq.md#how-does-the-jar-command-support-spring-boot-jars)). We are not planning to make `jib jar` more complex, so setting entrypoints will remain unsupported. `jib build` is the right choice for more complex logic.

ok, thans for your reply.
